### PR TITLE
Add description on how to use pre-commit and ts_pre_commit_conf.

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

**Comment by tribeiro** [comment]
There's something weird about this PR, I only see one change done in ``doc/procedures/index.rst`` but no real content. Also, I think you need to rebase your branch to ``main``.

**Comment by wvreeven** [comment]
Sorry, I forgot to add the new content. That's done now as well as the rebase.

File: doc/procedures/pre_commit.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+``ts_pre_commit_conf`` is a tool to generate configuration files for the pre-commit hooks for the Telescope and Site SoftWare (TSSW) team at the Vera C. Rubin Observatory.
+
+These configuration files facilitate ``pre-commit`` to maintain ``black`` formatting, ``flake8`` compliance and ``isort`` imports sorting.
+If ``mypy`` is used, a configuration file can be generated for that as well.
+The configuration files are generated by using the ``generate_pre_commit_conf`` command.
+Since TSSW uses ``conda`` (or ``mamba``) for their software installations, a conda recipe was created for this project and the conda package was made available in the ``lsstts`` conda channel.
+To enable this:
+
+* Install ``ts_pre_commit_conf``, for instance ``conda install -y -c lsstts ts-pre-commit-conf``.
```
Comment (tribeiro) [inline doc/procedures/pre_commit.rst:26]: I think the name of the package is actually `ts-pre-commit-config` (https://anaconda.org/lsstts/ts-pre-commit-config)...

**Review by tribeiro — APPROVED** [review]
Looks good.. please, just fix the name of the package before merging..

**Pull request merged**